mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.cpp

```diff
@@ -365,27 +383,37 @@ void CLeaderboardsTimes::LoadLocalTimes(KeyValues* kv)
             char pReplayPath[MAX_PATH];
             V_ComposeFileName(RECORDING_PATH, pFoundFile, pReplayPath, MAX_PATH);
 
-            CMomReplayBase *pBase = g_ReplayFactory.LoadReplayFile(pReplayPath, false);
-            Assert(pBase != nullptr);
-
-            if (pBase)
-                m_vLocalTimes.InsertNoSort(pBase);
+            FileAsyncRequest_t request;
+            request.pszFilename = pReplayPath;
+            request.pContext = this;
+            request.pfnCallback = LocalTimesCallbackForwarder;
+            request.pszPathID = "MOD";
+            filesystem->AsyncRead(request);
```

Comment:
Minor, but looks like this is expecting 2 arguments:
![image](https://user-images.githubusercontent.com/9014762/96934577-4cf94380-1477-11eb-877b-0f0175067e57.png)

Reply:
Yeah, if I am going to store the handles like I said above, this will be fixed